covid19-etl/etl/idph.py

```diff
@@ -59,18 +24,38 @@ def format_summary_report_submitter_id(location_submitter_id, date):
     return "{}_{}".format(location_submitter_id.replace("summary_location_", "summary_report_"), date)
 
 
-class IllinoisDPHETL:
-    def __init__(self, access_token):
+class IDPH(base.BaseETL):
+    def __init__(self, base_url, access_token):
+        super().__init__(base_url, access_token)
+
+        self.program_name = "open"
+        self.project_code = "IDPH"
+        self.metadata_helper = MetadataHelper(base_url=self.base_url,
+                                              program_name=self.program_name,
+                                              project_code=self.project_code,
+                                              access_token=access_token)
+
+        self.county_dict = {}
+
         self.summary_locations = []
         self.summary_reports = []
-        self.metadata_helper = MetadataHelper(access_token=access_token)
+
+    def il_counties(self):
+        with open("IL_counties_central_coords_lat_long.tsv") as f:
```

Comment:
i think you deleted this file instead of moving it

Reply:
True, committed again.